plugins/alerting/pkg/alerting/admin.go

```diff
@@ -218,7 +219,9 @@ func (p *Plugin) Info(ctx context.Context, _ *emptypb.Empty) (*alertops.Componen
 	if err != nil {
 		return nil, status.Error(codes.Unavailable, err.Error())
 	}
+	p.syncController.hashMu.Lock()
```

Comment:
Not a good idea to use the mutex from here, suggest doing something like `p.syncController.GetHash()` so that the sync controller can manage its own mutex; or, move the mutex into `Plugin`.

Reply:
this is a stop-gap fix for something else

Reply:
https://github.com/rancher/opni/pull/1672